fr/views.rst

```diff
@@ -567,44 +567,10 @@ fournissez une valeur unique de la clé cache en utilisant le format suivant::
         ]
     );
 
-Vous pouvez tirer profit des elements en utilisant ``requestAction()``. La
-fonction ``requestAction()`` récupère les variables de vues à partir
-d'une action d'un controller et les retourne en tableau. Cela permet à vos
-elements de fonctionner dans un style MVC pur. Créez une action du controller
-qui prépare les variables de la vue pour vos elements, ensuite appelez
-``requestAction()`` depuis l'intérieur du deuxième paramètre de ``element()``
-pour alimenter en variables de vues l'element depuis votre controller.
-
-Pour ce faire, ajoutez quelque chose comme ce qui suit dans votre controller,
-en reprenant l'exemple du Post::
-
-    class PostsController extends AppController
-    {
-        // ...
-        public function index()
-        {
-            $posts = $this->paginate();
-            if ($this->request->is('requested')) {
-                return $posts;
-            } else {
-                $this->set('posts', $posts);
-            }
-        }
-    }
-
-Et ensuite dans l'element, nous pouvons accéder au model des posts paginés.
-Pour obtenir les cinq derniers posts dans une liste ordonnée, nous ferions
-ce qui suit:
-
-.. code-block:: php
-
-    <h2>Latest Posts</h2>
-    <?php $posts = $this->requestAction('posts/index?sort=created&direction=asc&limit=5'); ?>
-    <ol>
-    <?php foreach ($posts as $post): ?>
-          <li><?= $post['Post']['title'] ?></li>
-    <?php endforeach; ?>
-    </ol>
+Si vous avez besoin de plus de logique dans votre element, comme des données
```

Comment:
Do we keep "element" or do we go with "élément" ?
These are the "edge" cases where I want to put the word in French but that still refers to a real concept in Cake...